config.example.wikimedia.yaml

```diff
@@ -152,34 +152,144 @@ templates:
           paths:
             /html/{title}:
               get:
+                x-setup-handler:
+                  - init:
+                      method: put
+                      uri: /{domain}/sys/key_rev_value/mobileapps.mobile-html
+                      body:
+                        revisionRetentionPolicy:
+                          type: 'latest'
+                          count: 1
+                          grace_ttl: 86400
+                        valueType: 'json'
+                        version: 1
                 x-request-handler:
-                  - get_from_backend:
+                  - handle_req:
                       request:
-                        uri: http://appservice.wmflabs.org/{domain}/v1/page/mobile-html/{title}
+                        method: get
+                        uri: /{domain}/sys/mobileapps/handling/mobile-html/{title}
+                        headers:
+                          cache-control: '{cache-control}'
```

Comment:
I that case, how is the storage going to be invalidated, if the render change is needed, but revision is the same (template change, or some other dependency change)